website/static/js/fangorn.js

```diff
@@ -833,6 +836,45 @@ function expandStateLoad(item) {
     reapplyTooltips();
 }
 
+/**
+ * @param tree A Treebeard _item object for the row
+ * @param nodeID Current node._id
+ * @param file window.contextVars.file object
+ */
+function findCurrentFileID(tree, nodeID, file) {
+    var tb = this;
+    if (tb.options.folderIndex !== undefined && tb.options.folderArray !== undefined && tb.options.folderIndex < tb.options.folderArray.length) {
```

Comment:
@sloria @caneruguz, yeah that seems to be the problem I'm running into. I tried setting `tb._fangorn` in `ondataload` but it is not available to `lazyLoadOnLoad`